src/App.tsx

```diff
@@ -18,28 +18,21 @@ import { Backdrop, CircularProgress, createMuiTheme, ThemeProvider } from '@mate
 import { SnackbarProvider } from 'notistack';
 import React from 'react';
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
-
+import { getConnectionChainInformation } from './configs/connectionChainInformation';
 import { light } from './components';
-import { CHAIN_1, CHAIN_2, getChainProviders } from './configs/substrateProviders';
 import { AccountContextProvider } from './contexts/AccountContextProvider';
 import { KeyringContextProvider } from './contexts/KeyringContextProvider';
 import { MessageContextProvider } from './contexts/MessageContext';
 import { SourceTargetContextProvider } from './contexts/SourceTargetContextProvider';
 import { TransactionContextProvider } from './contexts/TransactionContext';
-import { useApiConnection } from './hooks/useApiConnection';
+import { useConnections } from './hooks/useConnections';
 import Main from './screens/Main';
 
-const providers = getChainProviders();
+const connectionDetailsOne = getConnectionChainInformation('1');
```

Comment:
I dont understand how this hardcoded things (including the getConnectionChainInformation) is of any use... you are calling a function with hardcoded value, which then calls another function (getSubstratePairConfigs) from another file to get some more hardcoded values based on a hardcoded string ('RialtoMillau') and construct a JSON. It would be easier if we had a simple Map in a file and feed the values straight from there...

Reply:
I still depend on the .env values for constructing the api connections basically. 
The purpose of  the hardcoded values ("1" & "2") is to process those connection details per chain in  an agnostic way, meaning that instead of calling  variables in the code `RialtoHasher` it has to be `Chain1Hasher` so instead we need to implement a new pair like Rococo<>Westend , Chain1 be initialized with Rococo and Chain2 with Westend.

Then the use of the .env variable  REACT_APP_PAIR is for determining setting at code level specific custom values for each pair/chain ( you can see it with millau custom hasher )

Reply:
Anyway is better to extract the "1" and "2" to a const